flu-with-behavior/README.md

```diff
@@ -90,18 +89,42 @@ This file contains the code that implements the social distancing sub-model that
 condition STAY_HOME {
     start_state = No
 
-    state No {}
+    state No {
+        attend()
+		wait()
+		next()
```

Comment:
Are `wait()` and `next()` required to be stated explicitly in FRED 8, or is this change stylistic?